File: Easy Challenges/0016 - Character Removal/Solution.java
import java.util.Scanner;

public class Solution {

    public static void main(String[] args) {
        Scanner reader = new Scanner(System.in);
        System.out.print("Enter the string: ");
        String strOne = reader.nextLine();
        System.out.print("Enter the characters to remove:");
        String strTwo = reader.nextLine();
        System.out.printf("Modified string: %s", removeCharacters(strOne, strTwo));
    }

    public static String removeCharacters(String strOne, String strTwo) {
        for (int i = 0; i < strTwo.length(); i++) {
            strOne = strOne.replaceAll(strTwo.substring(i, i + 1), "");
        }
        return strOne;
    }

}

File: Easy Challenges/0016 - Character Removal/Solution.py
def remove_characters(str_one, str_two):
    for c in str_two:
        str_one = str_one.replace(c, "")
    return str_one


def main():
    str_one = str(input("Enter the string: "))
    characters = str(input("Enter characters to remove: "))
    print("Modified string: %s" % remove_characters(str_one, characters))

    
if __name__ == '__main__':
    main()
